src/main/kotlin/no/nav/syfo/personstatus/PersonoversiktStatusService.kt

```diff
@@ -35,6 +41,27 @@ class PersonoversiktStatusService(
         }
     }
 
+    suspend fun getAktiveVurderinger(
```

Comment:
Kanskje `getAktiveArbeidsuforhetVurderinger`?

Reply:
Tanken var at denne kunne gjelde generelt. Slik at man etterhvert kan legge inn for eksempel henting av status for en aktivitetskrav vurdering her 😊
Formålet med `getAktiveVurdering` var å oppdatere listen av `PersonOversiktStatus` med aktive vurderinger og legge de inn i dto objektet `PersonOversiktStatusDTO`.
Den renamingen til `getAktiveVurderinger` kan man selvfølgelig gjøre når man kommer til det punktet om vi ønsker det